.github/ISSUE_TEMPLATE/ios_background_message_handling.yml

```
@@ -0,0 +1,170 @@
+      options:
+        - label: I understand that iOS and android background messages behaves differently, and I've designed my application with that in mind.
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Do you have an active Apple Developer account?
+      description: |
+        An [Apple Developer account](https://developer.apple.com/membercenter/index.action) is required.
+      options:
+        - label: I have an active Apple Developer account.
+          required: true
+
+  - type: checkboxes
+    attributes:
+      label: Are you using a physical iOS device to test background messages?
+      description: |
+        A physical device is required, background messages will not work on a simulator.
+      options:
+        - label: I am using a physical iOS device to test background messages.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Have you enabled "Remote Notifications" & "Background Mode" (Checking options for "Background Processing" & "Remote Notifications") in your app's Xcode project?
+      description: |
+        Find out how to enable "Remote Notifications" & "Background Mode" in the [documentation](https://firebase.google.com/docs/cloud-messaging/flutter/client#enable_app_capabilities_in_xcode).
+
+        <details>
+        <summary>Click to see how your Xcode project "Signing and Capabilities" settings should look like</summary>
+        <img alt="Xcode "Signing and Capabilities tab" src="./../images/remote-notifications-enable.png?raw=true" />
+        </details>
+
+        REQUIRED: Please provide a screenshot of your Xcode project "Signing and capabilities" settings in the text area.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Have you created an APNs key in your Apple Developer account & uploaded this APNs key to your Firebase console?
+      description: |
+        You have to create an APNs key in your [Apple Developer account](https://developer.apple.com/membercenter/index.action) & upload this APNs key to your [Firebase console](https://console.firebase.google.com). [See documentation](https://firebase.google.com/docs/cloud-messaging/flutter/client#upload_your_apns_authentication_key).
+
+        <details>
+        <summary>Click to see how an APNs key uploaded to the correct app in the Firebase console should look like</summary>
+        <img alt="APNs key uploaded to Firebase console" src="./../images/apns-key-upload.png?raw=true" />
+        </details>
+
+        REQUIRED: Please provide a screenshot of your APNs key uploaded to your Firebase console to the correct app in the text area.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Have you enabled method swizzling in your app?
+      description: |
+        Go to your "Info.plist" file and add the following key `FirebaseAppDelegateProxyEnabled` with a value of `YES`.
```

```suggestion
        Go to your "Info.plist" file and add the following key `FirebaseAppDelegateProxyEnabled` with a value of `YES`.
```

This isn't required afaik, and it's not documented anywhere is it? The only time it's bad is if they've read some other external docs somewhere that have told them to add it and set it to `NO` - should we just ask for their Info.plist file instead and we can determine that?

---

Yeah, it's not needed unless disabled. Should've kept it like how it's described in the [previous documentation](https://firebase.flutter.dev/docs/messaging/overview#4-apple-integration). Will update to say something like "have you disabled method swizzling?"